bin/cluster_sce.R

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env Rscript
+
+# Script used to perform clustering on a given SCE object
+# 
+# This script reads in an RDS file containing an SCE, and performed
+#  graph-based clustering using the specified algorithm. Cluster identities are 
+#  stored in the SCE's colData slot, and the SCE is written back out to the
+#  original RDS file.
+
+# import libraries
+library(optparse)
+library(SingleCellExperiment)
+
+# Set up optparse options
+option_list <- list(
+  make_option(
+    opt_str = c("-i", "--processed_sce_file"),
```

Comment:
```suggestion
    opt_str = c("-i", "--input_sce_file"),
```
I think we want to be consistent with these scripts on how we name the input files. We've been using this pretty consistently.